mlflow/pipelines/utils/step.py

```diff
@@ -81,6 +81,12 @@ def display_html(html_data: str = None, html_file_path: str = None) -> None:
             subprocess.run(["open", html_file_path], check=True)
 
 
+# Prevent pandas_profiling from using multiprocessing on Windows while running tests.
+# multiprocessing and pytest doesn't play well together.
```

Comment:
Could you create a ticket on pandas profiling OSS project ?
Once they address it we can remove this.

Reply:
Is this a pandas_profiling issue?

Reply:
I don't know, but pandas profiling code triggers the error, it might has some relationship